src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/Safeguarding.cshtml

```diff
@@ -0,0 +1,68 @@
+﻿@page
+@model FamilyHubs.Referral.Web.Pages.ProfessionalReferral.SafeguardingModel
+@{
+}
+
+<style>
+    a.nhs:link {
+        color: white;
+        background-color: transparent;
+        text-decoration: none;
+    }
+
+    a.nhs:visited {
+        color: white;
+        background-color: transparent;
+        text-decoration: none;
+    }
+
+    a.nhs:hover {
+        color: white;
+        background-color: transparent;
+        text-decoration: underline;
+    }
+
+    a.nhs:active {
+        color: black;
+        background-color: transparent;
+        text-decoration: underline;
+    }
+
+    .interrupt-panel {
+        width: 100%;
+        padding: 30px;
+        background: #005ea5;
+        color: #fff !important;
+        overflow: hidden;
+        width: auto;
+    }
+
+        .interrupt-panel h1, .interrupt-panel p, .interrupt-panel h2, .interrupt-panel ul {
+            color: #fff !important;
+        }
+
+</style>
+
+<a asp-page="/ProfessionalReferral/LocalOfferDetail" asp-route-id="@Guid.NewGuid().ToString()" asp-route-serviceId="@Model.Id" asp-route-name="@Model.Name" class="govuk-back-link">Back</a>
```

Comment:
The Guid route-id looks a little strange to me.
I can see you've added a comment to the OnGetAsync()...
`//Needs to pass dummy id so service id can be any string`.
Can you please explain we why need it?